src/CloudFormation/Tests/Unit/Input/DescribeStackEventsInputTest.php

```diff
@@ -3,7 +3,7 @@
 namespace AsyncAws\CloudFormation\Tests\Unit\Input;
 
 use AsyncAws\CloudFormation\Input\DescribeStackEventsInput;
-use PHPUnit\Framework\TestCase;
+use AsyncAws\Core\Test\TestCase;
```

Comment:
not compatible with core 0.2 . provide a migration path.

Reply:
Core 0.2 is not released yet. So this is fine